app/views/events/participants.html.erb

```diff
@@ -1,40 +1,66 @@
 <%- model_class = Event -%>
+<script type="text/javascript">
```

Comment:
Putting them there creates a problem with turbolinks (code doesn't work when loading the page for the first time).